### ISSUE TITLE
Keep track of tag start/end positions

### ISSUE THREAD

**Issue by rolandpoulter** [namuol/cod#5]
It would be nice if the tags remembered what index they started and ended in the source. Then it would be easy to show the source code of tags.

**Comment by rolandpoulter** [comment]
I didn't realize that this creates a single object for the whole file.

Maybe if there was a way to tell it to return the current line/index of a file or something.

```
/**
  @$start
  @$end
*/
```

Could return:
`{start: 0, end: 20}`

**Comment by namuol** [comment]
Yeah this is probably a feature that many people will eventually want.

If cod supports file+line-numbers at all, it should be done transparently with a parameter, or maybe through some kind of plugin system.
